runtime/lua/vim/highlight.lua

```diff
@@ -0,0 +1,41 @@
+local api = vim.api
+
+local highlight = {}
+
+--- Highlight the yanked region
+--
+--- use from init.vim via
+---   au TextYankPost * lua require'vim.highlight'.on_yank()
+--- customize highlight group and timeout via
+---   au TextYankPost * lua require'vim.highlight'.on_yank("IncSearch", 500)
+-- @param higroup highlight group for yanked region
+-- @param timeout time in ms before highlight is cleared
+-- @param event event structure
```

Comment:
Would you add `@return ...` doc comments?

Reply:
But this function doesn't return anything?